orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -232,4 +236,32 @@ class OperationsController {
     def pipeline = orchestrationLauncher.start(json)
     [ref: "/tasks/${pipeline.id}".toString()]
   }
+
+  @RequestMapping(value = "/instance/enabled", method = RequestMethod.POST)
```

Comment:
Thinking to the future ... any value in rooting these sort of admin-like operations under a `/admin` path?

(I wouldn't be surprised if someday we support manipulating queue throttles via an admin API as well)

Might make it easier to specifically attach additional auth configurations (for example).